dbusmock/templates/networkmanager.py

```diff
@@ -26,8 +26,9 @@
 
 
 BUS_NAME = 'org.freedesktop.NetworkManager'
-MAIN_OBJ = '/org/freedesktop/NetworkManager'
-MAIN_IFACE = 'org.freedesktop.NetworkManager'
+MAIN_OBJ = '/org/freedesktop'
```

Comment:
I'm not sure about this. The object manager's path is /org/freedesktop because it needs to manage the manager object (/o/fd/NetworkManager) and thus needs to be above it.

I'm not familiar with dbusmock enough to know what properties are required to be kept stable for backwards compatibility. What change here is problematic, is it the change of MAIN_OBJ property?